packages/core/src/internal/content/rules/solidity/solidity-tests.md

```
@@ -0,0 +1,31 @@
+-   Use vm.bound for multiple fuzz case exclusions
+-   Use vm.assume only for single case exclusions
+-   Use vm.skip(true) for unimplemented tests
+-   Never comment out tests or leave empty bodies
+-   Use test coverage tools and aim for high test coverage, especially for critical paths.
+-   Follow BTT (Branching-Tree-Technique) pattern, first defining all the branches to be tested in the .tree in a format compatible with bulloak, and then writing the actual Unit test.
+
+## Integration Tests
+
+-   Fork relevant networks for external contract interactions
+-   Use createSelectFork with appropriate parameters:
+    -   `string calldata urlOrAlias, uint256 block`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)